wolfSSL/src/chapter02.md

```diff
@@ -472,6 +588,54 @@ Turns on the use of DTLS, or datagram TLS. This isn't widely supported or used.
 
 Turns on wolfSSL’s RSA key generation functionality. See [Keys and Certificates](chapter07.md#keys-and-certificates) for more information.
 
+#### WOLF_PRIVATE_KEY_ID
+
+FIPS v5 and older doesn't support WOLF_PRIVATE_KEY_ID with PK callbacks. Required for client authentification in some cases.
```

Comment:
This is used with PKCS11 to enable support for key id and label API's. The statement about FIPS v5 or older is correct. This is only required for PKCS11, but is also enabled with PK and Crypto callbacks.